docs/topics/proc_debug-start-native-debug-nodejs.adoc

```diff
@@ -0,0 +1,30 @@
+= Starting your Application Locally and Attaching the Native Debugger
+The native debugger enables you to debug your {NodeJS}&#x2013;based application using the built-in debugging client.
+
+.Prerequisites 
+* An application you want to debug.
+
+.Procedure
+. Start the application with the debugger enabled.
++
+The native debugger is automatically attached and provides a debugging prompt.
++
+.Example application with the debugger enabled
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ node inspect app.js
+< Debugger listening on ws://127.0.0.1:9229/12345678-aaaa-bbbb-cccc-0123456789ab
+< For help see https://nodejs.org/en/docs/inspector
+< Debugger attached.
+...
+debug>
+----
++
+If you have an link:https://expressjs.com[express] application, such as the xref:mission-http-api-nodejs[{name-mission-http-api} booster], starting the debugger is slightly different:
```

Comment:
I wonder if we should mention if you create the express application using the express-generator, https://expressjs.com/en/starter/generator.html

that is what creates the `./bin/www`   but you are referencing the http booster,  i'll leave it up to you :)

Reply:
Yea I think we should rather describe that what you need to do is, in general, `node inspect path/to/app/entrypoint`, rather than selectively choosing one particular web framework and saying that it's slightly different, while the difference is just the location of the entrypoint.